test/unit/lib/plugins/package/lib/packageService.test.js

```diff
@@ -466,82 +107,67 @@ describe('lib/plugins/package/lib/packageService.test.js', () => {
       });
     });
 
-    it.skip('TODO: should exclude default plugins localPath', () => {
-      // Confirm ".serverless-plugins/index.js" is not packaged
-      //
-      // Replaces
-      // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L87-L96
+    it('should exclude default plugins localPath', () => {
+      expect(fnIndividualZippedFiles).to.not.include('.serverless-plugins/index.js');
```

Comment:
This tests should technically confirm _service_ (not function specific package).

I've realized that some comments and setup were a bit misleading in this file, and I've just improved that in master with https://github.com/serverless/serverless/pull/8822

Please see the updates. I know that it now introduces conflicts in your PR which now requires an update. Sorry about that! If you feel it's too much, it's totally understood, let us know, and we will finish this PR then

Reply:
@medikoo I'll rebase with upstream and adjust tests accordingly.

I guess the next issue I'll take, would be related to a less active area than packaging 😅